tests/end-to-end/upgrade/main.go

```diff
@@ -5,6 +5,10 @@ import (
 	"fmt"
 	"time"
 
+	ebClientSet "github.com/kyma-project/kyma/components/event-bus/client/generated/clientset/internalclientset"
```

Comment:
Imports should always [remain lowercase](https://rakyll.org/style-packages/#lowercase-only).